lisa/runners/lisa_runner.py

```diff
@@ -687,6 +687,35 @@ def _merge_test_requirements(
                             schema.NodeSpace, node_requirement_data
                         )
 
+                        # Manage the union of the platform requirements and the node
+                        # requirements before taking the intersection of
+                        # the rest of the requirements.
+                        platform_requirement.features = search_space.SetSpace(
+                            True,
+                            (
+                                platform_requirement.features.items
+                                if platform_requirement.features
+                                else []
+                            )
+                            + (
+                                original_node_requirement.features.items
+                                if original_node_requirement.features
+                                else []
+                            ),
+                        )
+                        platform_requirement.excluded_features = search_space.SetSpace(
+                            False,
```

Comment:
why False here? if I am reading it right, if there any excluded features then we would set it as true. isn't it?

Reply:
Sorry, just found the comments after merged.  The excluded features will be converted to True when get min capabilities.